js/components/oppfolgingsdialog/godkjennplan/releasetplan/DelingIkon.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { CSSTransition } from 'react-transition-group';
+import { getContextRoot } from '../../../../routers/paths';
+
+const IconAndText = styled.div`
+    margin: 1em 1em 0 0;
+    display: flex;
+`;
+
+const Text = styled.p`
+    margin: 0 .5em;
+`;
+
+const Image = styled.img`
+    width: 1.5em;
+`;
+
+const fadeOutDelay = 6000;
+const DOMTimeoutDelay = 4000;
+
+class DelingIkon extends React.Component {
```

Comment:
Kan bruke hooks i stedet for dette naa:
UseEffect i stedet for this.state og useEffect i stedet for componentDidMount. Kan evt. vise.

Reply:
👍

Reply:
@janzh Fikset 😄